nservicebus/rabbitmq/rabbitmqclient-nuget.md

```diff
@@ -0,0 +1,32 @@
+---
+title: RabbitMQ.Client Nuget usage
+summary: Explains why the RabbitMQ.Client Nuget version range is inconsistent with most other NServiceBus extension nugets.
+---
+
+The NServiceBus.RabbitMQ library uses the [RabbitMQ.Client](http://www.nuget.org/packages/RabbitMQ.Client/) to communicate with the underlying queueing system. Due to the API/nuget versioning strategy of RabbitMQ.Client there are some complexities to targeting that library from the perspective of in intermediary library like NServiceBus.RabbitMQ.
+
+Most libraries NServiceBus integrates with follow [SemVer](http://semver.org/). This means that those libraries do not make breaking changes in Minor or Patch releases. So the nuget range for these libraries is generally `≥ CurrentMajor.0.0 && < NextMajor.0.0`. So it is safe for the consumer of these libraries to move between any Minor of the current Major version.
+
+The RabbitMQ.Client library does not follow SemVer. This means they are free to make breaking changes in Major, Minor or Patch released. 
+
+For example between versions 3.5.1 and 3.5.2 of RabbitMQ.Client the public type `RabbitMQ.Client.Framing.Impl.Connection` had two public fields remove and 3 public methods removed (luckily none of these effect the API surface area used by NServiceBus.RabbitMQ). 

```

Comment:
typo - removed not remove, and affect not effect. Affect is a verb and effect is a noun. 

`For example between versions 3.5.1 and 3.5.2 of RabbitMQ.Client the public type `RabbitMQ.Client.Framing.Impl.Connection` had two public fields and three public methods removed (luckily none of these affect the API surface area used by NServiceBus.RabbitMQ).`